src/dto/Validator.java

```diff
@@ -0,0 +1,66 @@
+package dto;
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+
+            public List<String> validate(Object object) throws IllegalAccessException {
```

Comment:
Cognitive Complexity of methods should not be too high․
Cognitive Complexity is a measure of how hard the control flow of a method is to understand. Methods with high Cognitive Complexity will be difficult to maintain.
It would be better to refactor this method.